<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stoking_p</class>
 <widget class="QMainWindow" name="stoking_p">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>stoking_p</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* GENERAL */
QWidget {
    background-color: #f4f6f8;
    font-family: &quot;Segoe UI&quot;, &quot;Arial&quot;;
    font-size: 14px;
    color: #333;
}

/* LABELS */
QLabel {
    color: #222;
    font-weight: 500;
}

/* LINE EDITS */
QLineEdit {
    border: 1px solid #ccc;
    border-radius: 5px;
    padding: 6px 8px;
    background-color: #fff;
}
QLineEdit:focus {
    border: 1px solid #0078d7;
}

/* BUTTONS */
QPushButton {
    background-color: #0078d7;
    color: white;
    padding: 6px 12px;
    border: none;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #005fa3;
}
QPushButton:pressed {
    background-color: #004f87;
}

/* TABLES */
QTableWidget {
    background-color: #ffffff;
    border: 1px solid #ccc;
    gridline-color: #ccc;
    selection-background-color: #d0e7ff;
    alternate-background-color: #f9f9f9;
}
QHeaderView::section {
    background-color: #e8eaec;
    padding: 6px;
    border: 1px solid #dcdcdc;
    font-weight: bold;
}

/* IMAGE PREVIEW LABEL (Assumes objectName = imageLabel) */
QLabel#imageLabel {
    border: 1px dashed #aaa;
    background-color: #fff;
    min-width: 100px;
    min-height: 100px;
}

/* SCROLLBARS (optional for table scrolling) */
QScrollBar:vertical, QScrollBar:horizontal {
    background: #e0e0e0;
    border: none;
    width: 10px;
}
QScrollBar::handle:vertical, QScrollBar::handle:horizontal {
    background: #aaa;
    border-radius: 5px;
}
QScrollBar::handle:hover {
    background: #888;
}
</string>
  </property>
  <widget class="QWidget" name="centralHolder">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>64</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>64</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="title_p">
         <property name="focusPolicy">
          <enum>Qt::FocusPolicy::WheelFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size: 21px;
font-weight: bold;
color: #0078d7;
</string>
         </property>
         <property name="text">
          <string>STOKING PRODUCTS</string>
         </property>
         <property name="textFormat">
          <enum>Qt::TextFormat::PlainText</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>1138</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QScrollArea" name="windowButtonsHolder">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>198</width>
           <height>652</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
font-size: 18px;
font-weight: bold;
color: #fff;
}</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QPushButton" name="ShoppingCart">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Shopping Cart</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="StoreItems">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Store Items</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ProductHistory">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>48</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string>Product History</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacerForBtns">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>350</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="windowHolder">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="shopingWindow">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="cartMainWindow">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="searchShop">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="addCartItem">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>Segoe UI</family>
                  <pointsize>-1</pointsize>
                  <bold>true</bold>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clearCart">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>CLEAR</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="cartListTB"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="transaction_window">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,2,0,1,0,16">
             <item>
              <widget class="QLineEdit" name="transactionNameLineEdit"/>
             </item>
             <item>
              <spacer name="verticalSpacer_6">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="summaryLabel">
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_5">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="add_transaction">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>48</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>48</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>FINISH TRANSACTION</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="itemsWindow">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="searchItem">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="addNewItem">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>32</height>
                 </size>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>ADD A NEW ITEM</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTableView" name="itemListTB"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QFrame" name="editor">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0,0,0,0,0,0,0,0,0,0,1,0,8">
             <property name="spacing">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="IncertMode_lb">
               <property name="text">
                <string>ADD MODE</string>
               </property>
               <property name="scaledContents">
                <bool>false</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
               <property name="wordWrap">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="itemName_lb">
               <property name="text">
                <string>Item Name:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="itemName_edit"/>
             </item>
             <item>
              <widget class="QLabel" name="itemType_lb">
               <property name="text">
                <string>Item Type:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="itemType_edit"/>
             </item>
             <item>
              <widget class="QLabel" name="itemPrice_lb">
               <property name="text">
                <string>Selling Price:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="itemPrice_edit"/>
             </item>
             <item>
              <widget class="QLabel" name="itemBuy_lb">
               <property name="text">
                <string>Buying Price:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="itemBuy_edit"/>
             </item>
             <item>
              <widget class="QLabel" name="itemCount_lb">
               <property name="text">
                <string>Stock Count:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="itemCount_edit">
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="addTableItem_btn">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>48</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>48</height>
                </size>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="text">
                <string>ADD ITEM</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="historyWindow">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTableView" name="historyTable"/>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="minimumSize">
             <size>
              <width>250</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>250</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Raised</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
